### DISCUSSION TITLE
绳网竟然有镜像站了

### DISCUSSION THREAD

**Discussion by somerui** [share121/inter-knot#2953] [其他]
这种灰产网站真的适合放在明面上吗...搞成洋葱那样的网站更安全吧

**Comment by NiuBoss123** [comment]
真的假的，发一个看看

**Comment by fancy-fantasy** [comment]
wow六级糕手绳匠<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Comment by user51534** [comment]
其实搞成匿名更好，参考死透的A岛<img src="https://img-static.mihoyo.com/communityweb/upload/73aabfbe4b49722db27bd5341431f58a.png" alt="丹恒 思考" width="75" >

**Reply by fashibuding1214** [reply]
现在是x岛了

**Comment by 1ctrl-cv** [comment]
洋葱的速度有点太慢了（

**Reply by rong-xiaoli** [reply]
慢是一点，另一点我觉得主要是洋葱比较复杂，像iOS得搞个orbot才能连洋葱站

**Reply by 1ctrl-cv** [reply]
> 慢是一点，另一点我觉得主要是洋葱比较复杂，像iOS得搞个orbot才能连洋葱站

那倒无所谓，越复杂越能防止不明生物爆破

**Reply by rong-xiaoli** [reply]
> > 慢是一点，另一点我觉得主要是洋葱比较复杂，像iOS得搞个orbot才能连洋葱站
> 
> 那倒无所谓，越复杂越能防止不明生物爆破

相信GayHub，人家可是全球最大~~同性交友~~代码托管平台。唯一需要担心的就是作者的算法了。要是有懂信息安全的能帮忙，那就再好不过了。

**Reply by 1ctrl-cv** [reply]
> > > 慢是一点，另一点我觉得主要是洋葱比较复杂，像iOS得搞个orbot才能连洋葱站
> > 
> > 
> > 那倒无所谓，越复杂越能防止不明生物爆破
> 
> 相信GayHub，人家可是全球最大~同性交友~代码托管平台。唯一需要担心的就是作者的算法了。要是有懂信息安全的能帮忙，那就再好不过了。

那确实

**Comment by somerui** [comment]
草，我只是玩梗模仿绳网的帖子
